docker/dev/ubuntu/scripts/examples/orion-dev.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+source $HOME/.bashrc
+docker --tlsverify=false run -p 1026:1026 --name orionDev --link mongoOrion:mongoOrion -v /Users/jmcf/work/develop/fiware-orion:/root/src -t -i orion-dev:latest /bin/bash
```

Comment:
`/Users/jmcf/work/develop/fiware-orion` is user-specific. Should be replaced in the documention, e.g. using some generic like  `/path/to/fiware-orion` or something like that.